cluster/network.go

```diff
@@ -70,6 +71,9 @@ const (
 
 	WeaveNetworkPlugin  = "weave"
 	WeaveNetworkAppName = "weave-net"
+
+	AciNetworkPlugin = "ACI"
```

Comment:
All the network plugins are lowercase and the convention in the code seems to be `Aci` which is also conform the use of `aci` here.

Reply:
ACI is an acronym(Application Centric Infrastructure), so it is capitalized. I will change the rest of the arguments to use the capitalized convention as well, hope this is ok..

Reply:
This is now changed back to "aci"